### PR TITLE
s/form state/action state

### PR DESCRIPTION
**Pull request by rickhanlonii**
Rename internals from "form state" to "action state"

### CODE REVIEW

File: packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js [hunk 1]
```diff
@@ -1819,12 +1819,14 @@ function injectFormReplayingRuntime(
 const formStateMarkerIsMatching = stringToPrecomputedChunk('<!--F!-->');
 const formStateMarkerIsNotMatching = stringToPrecomputedChunk('<!--F-->');
 
+// TODO: rename to pushActionState?
```
Comment (rickhanlonii) [inline packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js:1822]: @acdlite can you confirm if these two functions should be `pushFormState*`?

File: packages/react-reconciler/src/ReactInternalTypes.js [hunk 2]
```diff
@@ -265,6 +265,7 @@ type BaseFiberRootProperties = {
     errorInfo: {digest?: ?string, componentStack?: ?string},
   ) => void,
 
+  // TODO: should this be called actionState?
```
Comment (rickhanlonii) [inline packages/react-reconciler/src/ReactInternalTypes.js:268]: @acdlite is this `ReactActionState` now?

File: packages/react-server/src/ReactFizzServer.js [hunk 3]
```diff
@@ -334,6 +334,7 @@ export opaque type Request = {
   // rendering - e.g. to the client. This is considered intentional and not an error.
   onPostpone: (reason: string, postponeInfo: ThrownInfo) => void,
   // Form state that was the result of an MPA submission, if it was provided.
+  // TODO: is this actionState?
```
Comment (rickhanlonii) [inline packages/react-server/src/ReactFizzServer.js:337]: @acdlite should this rename to `actionState`?

File: packages/react-server/src/ReactFizzServer.js [hunk 4]
```diff
@@ -4266,6 +4267,7 @@ export function flushResources(request: Request): void {
   enqueueFlush(request);
 }
 
+// TODO: is this action state, or specific to forms?
```
Comment (rickhanlonii) [inline packages/react-server/src/ReactFizzServer.js:4270]: Ditto here, is this `getActionState`?

File: packages/react-server/src/ReactFizzServer.js (hunk 3 above)
Comment (sebmarkbage) [reply packages/react-server/src/ReactFizzServer.js:337]: This is specific to forms. It's the progressive enhancement feature.

Comment (rickhanlonii) [reply packages/react-server/src/ReactFizzServer.js:337]: I thought that might be the case, thank for confirming. I'll remove the TODOs and give it another pass.

**Review by kassens — APPROVED** [review]

**Pull request merged**